### PR TITLE
Add support for symlinks in PlexusIoZipFileResourceCollection

### PR DESCRIPTION
**Pull request by plamentotev**
Looks like I forgot to add fix for `PlexusIoZipFileResourceCollection` in #11. It looks like one cannot update pull request once it's merged so I'm making new one.

### CODE REVIEW

File: src/main/java/org/codehaus/plexus/archiver/zip/PlexusIoZipFileResourceCollection.java [hunk 1]
```diff
@@ -79,6 +80,59 @@ public URL getResource( String name )
     private static class ZipFileResourceIterator
         implements Iterator<PlexusIoResource>, Closeable
     {
+        private class ZipFileResource
+            extends PlexusIoURLResource
+        {
+            private ZipFileResource( ZipArchiveEntry entry )
+            {
+                super( entry.getName(), entry.getTime() == 1 ? PlexusIoResource.UNKNOWN_MODIFICATION_DATE : entry.getTime(),
```
Comment (krosenvold) [inline src/main/java/org/codehaus/plexus/archiver/zip/PlexusIoZipFileResourceCollection.java]: Didnt you change the condition here ?

Comment (plamentotev) [inline src/main/java/org/codehaus/plexus/archiver/zip/PlexusIoZipFileResourceCollection.java]: Indeed I have. I should be `entry.getTime() == -1` I'll fix it

Comment (plamentotev) [inline src/main/java/org/codehaus/plexus/archiver/zip/PlexusIoZipFileResourceCollection.java]: Now it should be correct :smile:

**Comment by krosenvold** [comment]
The duplication of this class is unfortunate, what's worse is that they have somewhat different logic. One of them used to be in plexus-io. Ubderstanding the difference between these two is a bit of detective work. I believe there is an IT IN maven-assembly-plugin that fails wheb one of these is simly merged into the other...

**Pull request merged**